rskj-core/src/main/java/co/rsk/rpc/modules/personal/PersonalModuleWalletEnabled.java

```diff
@@ -63,7 +64,7 @@ public void init(RskSystemProperties properties) {
         // wallets in production for security reasons.
         Account coinbaseAccount = properties.localCoinbaseAccount();
         if (coinbaseAccount != null) {
-            this.wallet.addAccount(coinbaseAccount);
+            this.wallet.addAccount(coinbaseAccount).getBytes();
```

Comment:
I think this `.getBytes()` call should be removed

Reply:
:+1: